### DISCUSSION TITLE
Any plans to support Google’s AlloyDB?

### DISCUSSION THREAD

**Discussion by reev-create** [directus/directus#14118] [Archive]
Hey everybody,
Directus is one of the best CMS out there. I'd love to see Google's AlloyDB to be incorporated with it.

**Comment by benhaynes** [comment]
Thank you!! AlloyDB is Postgres-compatible, so it should work **now**. That said, we don't (can't) run tests on absolutely every vendor, so hopefully you or the community can give us some testing insight!

**Reply by reev-create** [reply]
Cool
